starter-code/src/functions-and-arrays.js

```diff
@@ -60,6 +101,18 @@ const wordsFind = [
   'disobedience'
 ];
 
+
+function doesWordExist(wordsFind,element){
+  for (var i = 0; i<wordsFind.length; i++){
+    if(wordsFind[i]===element) {
+      return true; 
+    }
+  }
+  return false; 
+}
+console.log(doesWordExist(wordsFind,trouble))
+
+
 // Counting Repetition
```

Comment:
Là aussi vous vous êtes un peu compliqué la vie (mais vous avez la méthode) : il suffit d'ajouter +1 à count à chaque fois que wordsCount[i] === à la string que vous avez défini en amont 
function howManyTimes(wordsCount, word) {
  var appearances = 0;
  if (wordsCount.length === 0) {
    return undefined;
  }
  for (var i = 0; i < wordsCount.length; i++) {
    if (wordsCount[i] === word) {
      appearances++;
    }
  }
  return appearances;
}